web_app/assets/styles.css

```diff
@@ -1,19 +1,135 @@
+/* Custom & Overriding CSS */
 
 
 /*------------------------------------------------------------------------------------*/
-/* Dropdown Menus */
+/* General/Misc. */
+
+.caps {
+    text-transform: uppercase;
+    letter-spacing: .075em;
+    font: 12px Arial;
+}
+
+
+/*------------------------------------------------------------------------------------*/
+/* Backgrounds/Bodies */
+
+.background {
+    padding-left: 5%;
+    padding-right: 5%;
+    background-color: lightgrey;
+}
+
+.content {
+    padding: 2em;
+    background-color: whitesmoke;
```

Comment:
I'm curious where `whitesmoke` is defined as a color.  I wasn't aware it was a browser built-in.

Reply:
It's builtin: https://www.w3schools.com/colors/colors_names.asp

Reply:
I remember a time when browsers only recognized 16 color names :smile:

Reply:
:+1: for some reason it's 140